kubernetes/annotations_and_labels.md

```diff
@@ -51,16 +47,38 @@ This page defines common annotations and labels we set in Kubernetes objects.
   value may be equal to that of `app.kubernetes.io/version` but it has a
   different purpose and since there could be multiple operators reconciling one
   object there could be multiple per-operator labels on one object.
-- `app.giantswarm.io/branch` - (informational) branch from which this
-  instance of the app/operator was built from.
+- `helm.sh/chart` - value should contain a chart name and version, see
+  [Helm chart labels best practices][helm-labels]
+- `app.kubernetes.io/name` - the name of the application; should be applied to
+  every Kubernetes resource associated with that application, i.e. all objects
+  in a chart that installs it.
+- `app.kubernetes.io/instance` - value should be set to `{{ .Release.Name }}`
+  and is meant for differentiating between instances of the same application.
+  This together with `app.kubernetes.io/name` should be used as replicas
+  selector, e.g.
+  `app.kubernetes.io/name=kvm-operator,app.kubernetes.io/instance=kvm-operator-1.0.0`.
+- `app.kubernetes.io/version` and other common labels (see
+  [here][helm-labels] and [here][k8s-common-labels]) - are informational,
```

Comment:
I would skip the other common labels part. This page is about labels we use and how we do this. If we want to use any of the common upstream labels it should be listed here.

Reply:
Sure, makes sense.

Reply:
Done